extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -383,26 +383,35 @@ class QueueProcessor extends EventEmitter {
     }
 
     scheduleResume(date) {
+        function triggerResume() {
+            this._updateZkStateNode('scheduledResume', null, err => {
+                if (err) {
+                    this.logger.error('error occurred saving state ' +
```

Comment:
Maybe add a log indicating it is being rescheduled for a minute later.

Reply:
I'll update the message to:
`error occurred saving state to zookeeper for resuming a scheduled resume. Retry again in 1 minute`